packages/account/src/Components/reset-trading-password-modal/reset-trading-password-modal.jsx

```diff
@@ -206,7 +206,7 @@ const ResetTradingPasswordModal = ({
     const [dialog_title, setDialogTitle] = React.useState('');
     const history = useHistory();
     React.useEffect(() => {
-        if (is_visible) {
+        if (is_visible && history.location.search !== '') {
```

Comment:
replacing the history resulted in rerendering the component , adding the condition prevent the unnecessary re-rendering